src/main/pg/rx_pwm.c

```diff
@@ -38,9 +38,18 @@ PG_REGISTER_WITH_RESET_FN(pwmConfig_t, pwmConfig, PG_PWM_CONFIG, 0);
 void pgResetFn_pwmConfig(pwmConfig_t *pwmConfig)
 {
     pwmConfig->inputFilteringMode = INPUT_FILTERING_DISABLED;
-    for (unsigned inputIndex = 0; inputIndex < PWM_INPUT_PORT_COUNT; inputIndex++) {
-        pwmConfig->ioTags[inputIndex] = timerioTagGetByUsage(TIM_USE_PWM, inputIndex);
-    }
+#ifdef RX_PWM1_PIN
+    pwmConfig->ioTags[0] = IO_TAG(RX_PWM1_PIN);
+#endif
+#ifdef RX_PWM2_PIN
+    pwmConfig->ioTags[1] = IO_TAG(RX_PWM2_PIN);
+#endif
+#ifdef RX_PWM3_PIN
+    pwmConfig->ioTags[2] = IO_TAG(RX_PWM3_PIN);
+#endif
+#ifdef RX_PWM4_PIN
+    pwmConfig->ioTags[3] = IO_TAG(RX_PWM4_PIN);
+#endif
```

Comment:
This could be done with a macro, although perhaps this is clearer.

Reply:
Yeah it's hard because this looks repetitive but is indeed very clear.